src/types.ts

```diff
@@ -39,8 +41,12 @@ export interface ColorModel<T extends AnyColor> {
   equal: (first: T, second: T) => boolean;
 }
 
-export interface ColorPickerBaseProps<T extends AnyColor> {
-  className: string;
+type ColorPickerHTMLAttributes = Omit<
+  React.HTMLAttributes<HTMLDivElement>,
+  "color" | "onChange" | "onChangeCapture"
```

Comment:
What's the reason for `"color"`? As far as I know no such attribute exists, so I can't imagine there's even any type collision there?

Reply:
I was super confused as well, but it's present in React's default types:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1785

Reply:
![image](https://user-images.githubusercontent.com/206567/110666551-7de5d500-81da-11eb-9226-80b3cfaaa1a0.png)

Reply:
I assume it's related to super-legacy stuff like `<font color="red">text</font>`.

Reply:
Huh, weird. Even in super old legacy stuff that shouldn't apply to an HTMLDivElement? Gotta question what those React folks are doing some days. 

But fair, makes sense